pkg/agent/controller/cleanup.go

```diff
@@ -48,17 +48,17 @@ var (
 func (a *Controller) Cleanup() error {
 	// Delete all ServiceImports from the local cluster skipping those in the broker namespace if the broker is on the
```

Comment:
there is no serviceImportSyncer anymore and unfortunately the serviceImportDownloader which is a resource syncer does not expose GetLocalClient() etc..
But I agree its a code smell. Did a refactoring to keep all clients and federators directly at the controller struct instead of taking them from one of the syncers.